llvm/include/llvm/Transforms/IPO/Attributor.h

```diff
@@ -6285,12 +6341,38 @@ struct AAAllocationInfo : public StateWrapper<BooleanState, AbstractAttribute> {
     return AbstractAttribute::isValidIRPositionForInit(A, IRP);
   }
 
+  // A helper function to check is simplified values exists for the current
+  // instruction.
+  bool simplifiedValuesExists(Attributor &A, Instruction *LocalInst) {
```

Comment:
```suggestion
  bool checkIfSimplifiedValuesExists(Attributor &A, Instruction *LocalInst) {
```

LLVM standard generally suggests the first word is a verb.